go.mod

```diff
@@ -12,4 +12,34 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/apimachinery v0.21.0
 	k8s.io/client-go v0.20.5
+	k8s.io/kops v1.19.0
+	sigs.k8s.io/controller-runtime v0.8.3
+)
+
+replace (
+	// Used to get around some weird etcd/grpc incompatibilty
+	google.golang.org/grpc => google.golang.org/grpc v1.29.0
+	// Used to pin the k8s library versions regardless of what other dependencies enforce
+	k8s.io/api => k8s.io/api v0.19.5
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.5
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.5
+	k8s.io/apiserver => k8s.io/apiserver v0.19.5
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.5
+	k8s.io/client-go => k8s.io/client-go v0.19.5
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.5
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.5
+	k8s.io/code-generator => k8s.io/code-generator v0.19.5
+	k8s.io/component-base => k8s.io/component-base v0.19.5
+	k8s.io/cri-api => k8s.io/cri-api v0.19.5
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.5
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.5
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.5
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.5
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.5
+	k8s.io/kubectl => k8s.io/kubectl v0.19.5
+	k8s.io/kubelet => k8s.io/kubelet v0.19.5
+	k8s.io/kubernetes => k8s.io/kubernetes v1.19.5
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.5
+	k8s.io/metrics => k8s.io/metrics v0.19.5
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.5
 )
```

Comment:
hacky rubbish for getting kops to work, copied from weaveworks/eksctl. Placeholder until @Callisto13's printer is in

Reply:
are you waiting for me to merge first?

Reply:
no, this can get merged first and then fixed later :smile: